physutils/physio.py

```diff
@@ -8,6 +8,7 @@
 
 import matplotlib.pyplot as plt
 import numpy as np
+import pandas as pd
```

Comment:
Would it be possible to drop this? Why is pandas' `isna` prefereable to numpy's `isnan`?

Reply:
There was an error thrown before, when initializing the fs as None, because the None type pydra handles is a `_Nothing` type not classic `None`.

But actually, now that I retested it, because another change was made in the task this is no longer needed, so I'll revert this